tests/libnode/node.go

```diff
@@ -380,3 +408,11 @@ func GetWorkerNodesWithCPUManagerEnabled(virtClient kubecli.KubevirtClient) []k8
 	Expect(nodeList).ToNot(BeNil())
 	return nodeList.Items
 }
+
+func SimulateUnhealthyNode(virtCli kubecli.KubevirtClient, nodeName string) {
```

Comment:
I would inline this, instead of adding this func.

Reply:
The function has been dropped.